### PR TITLE
add session example

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**

### CODE REVIEW

File: sessions/session_store.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/cloud/firestore"
+require "rack/session/abstract/id"
+
+module Rack
+  module Session
+    class FirestoreSession < Abstract::Persisted
```
Comment (bshaffer) [inline sessions/session_store.rb]: This is great... should we add this to `google-cloud-ruby-firestore`?

File: sessions/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Session handling with Firestore
+
+* A short App Engine app, which uses [Firestore](https://cloud.google.com/firestore/) to serve a webpage saying `"#{session['views']} views for #{session['greeting']}"`. Here, `session["greeting"]` is one of `["Hello World", "Hallo Welt", "Ciao Mondo", "Salut le Monde", "Hola Mundo"]` which has been randomly assigned to the viewer, and `session["views"]` is the number of times that viewer has loaded the page.
+* To run, follow Ruby's [Handling sessions with Cloud Firestore](https://cloud.google.com/ruby/getting-started/session-handling-with-firestore) tutorial.
```
Comment (bshaffer) [inline sessions/README.md]: We should wait to merge this until [the above link](https://cloud.google.com/ruby/getting-started/session-handling-with-firestore) does not 404

File: sessions/session_store.rb (hunk 1 above)
Comment (TheRoyalTnetennba) [reply sessions/session_store.rb]: My gut instinct is that it'd have to be a separate gem, and it would need some refining

Comment (bshaffer) [reply sessions/session_store.rb]: sgtm

**Pull request merged**